browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -3308,7 +3308,7 @@ index f52e4abc8e76c02285b9c68ac9c7a98bd203e0dd..a3f08d8c0644ac2ca61c66c979d3ae7a
 +            return makeUnexpected("Unable to decode given content"_s);
 +
 +        ScriptExecutionContext* context = element->scriptExecutionContext();
-+        fileObjects.append(File::create(context, Blob::create(context, SharedBuffer::create(WTFMove(*buffer)), type), name));
++	fileObjects.append(File::create(context, Blob::create(context, Vector { buffer->data(), buffer->size() }, type), name));
```

Comment:
Just `Blob::create(context, WTFMove(*buffer))` to avoid extra copy?

Reply:
Let's address it in follow-up, I'm going to land this PR and see if it builds on all platforms.